{"name": "Вход с неверным логином и паролем", "status": "passed", "start": 1729621201409, "stop": 1729621201698, "uuid": "e1284ce2-8d2c-4b97-b725-6b9662968985", "historyId": "0434c65fa756c765914bfdec2b43cb52", "testCaseId": "0434c65fa756c765914bfdec2b43cb52", "fullName": "tests.test_login_user.TestLoginUser#test_login_with_incorrect_login_and_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "User-PC"}, {"name": "thread", "value": "8308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}